tests/cnf/ran/gitopsztp/internal/helper/cluster.go

```diff
@@ -0,0 +1,18 @@
+package helper
+
+import (
+	"github.com/openshift-kni/eco-goinfra/pkg/clients"
+	"github.com/openshift-kni/eco-goinfra/pkg/ocm"
+)
+
+// DoesClusterLabelExist looks for a label on a managed cluster and returns true if it exists.
+func DoesClusterLabelExist(client *clients.Settings, clusterName string, label string) (bool, error) {
```

Comment:
could you move this to test/cnf/ran/internal/rancluster please? I think it could be useful for multiple test suites so good to have it in a more visible place